### PR TITLE
[CDAP-8074] - Adds Overview Top Section

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Adds `Tags, Description & Versions` components
- Adds `OverviewMetaSection` that contains the version, fast actions for an entity 

#### To do:
- Enhance `Description` component to dynamically detect string length and show More button to show full description
- Enhance `Tags` component to have the ability to add & remove tags and have the same More button to show all tags.

Bamboo build: http://builds.cask.co/browse/CDAP-DRC5421-1

### CODE REVIEW

File: cdap-ui/app/cdap/components/Description/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      description: this.props.description
+    };
+  }
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.description !== this.state.description) {
+      this.setState({
+        description: nextProps.description
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Description/index.js:31]: wouldn't displaying `props.description` will do this anyways?

File: cdap-ui/app/cdap/components/Overview/OverviewMetaSection/index.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      !isNil(entity) &&
+      entity.id !== objectQuery(this.state, 'entity', 'id')
+    ) {
+      this.setState({
+        entity
+      });
+    }
+  }
+  onFastActionsUpdate() {
+    console.log('Fast actions updated');
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Overview/OverviewMetaSection/index.js]: remove?

File: cdap-ui/app/cdap/components/Overview/OverviewMetaSection/index.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+              <TimeAgo date={parseInt(creationTime, 10)} />
+            </small>
+          </div>
+          <FastActions
+            className="overview-fast-actions"
+            entity={this.props.entity}
+            onUpdate={this.onFastActionsUpdate.bind(this)}
+          />
+        </div>
+        <Description description={description}/>
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/Overview/OverviewMetaSection/index.js]: nitpicking: add a space before the closing tag?

File: cdap-ui/app/cdap/components/Description/index.js (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/Description/index.js:31]: Talked to @elfenheart We will remove this when we add loading animation on entity switch

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**